index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+  </head>
+  <content>
+  <body id="body">
+
+    <ul id="FirstAndPike">
+      <h1> FirstAndPike </h1>
+    </ul>
+    <ul id="seaTac">
+      <h1> seaTac </h1>
+    </ul>
+    <ul id="seattleCenter">
+      <h1> seattleCenter </h1>
+    </ul>
+  </ul>
+  <ul id="capitolHill">
+    <h1> capitolHill </h1>
+  </ul>
+</ul>
+<ul id="alki">
```

Comment:
Be careful about your indentation! All of these uls are siblings, so they should be nested equally deeply.